edx_django_utils/hooks/pipeline.py

```diff
@@ -0,0 +1,83 @@
+"""
+TODO: add description
+"""
+from logging import getLogger
+
+from .exceptions import HookException
+from .utils import get_functions_for_pipeline, get_pipeline_configuration
+
+log = getLogger(__name__)
+
+
+def run_filter_pipeline(trigger_name, *args, **kwargs):
+    """
+    Function that manages the filters pipeline.
+
+    Arguments:
+        trigger_name:
+
+    Returns:
+        result:
+    """
+    pipeline, is_async = get_pipeline_configuration(trigger_name)
```

Comment:
I added `get_pipeline_configuration` to use it in `run_filter_pipeline` and `run_action_pipeline` because they use the same configurations, do we need to consider that each pipeline would have its own configuration? For example,

setting1 is just used by filter pipeline
setting2 is just used by action pipeline

If we consider this, I'll remove this helper function @morenol

Reply:
Right now I don't see any problem in having the same configuration for both pipelines.